.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: python
 jobs:
   include:
-    - name: "Python 3.8.0 on Xenial Linux"
```

Comment:
Thanks for catching this. I think GitHub actions is good enough, I probably will delete this config later.